cinema_service/urls.py

```diff
@@ -2,10 +2,17 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView, SpectacularRedocView
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
-    path("api/cinema/", include("cinema.urls", namespace="cinema")),
-    path("api/user/", include("user.urls", namespace="user")),
-    path("__debug__/", include("debug_toolbar.urls")),
-] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+                  path("admin/", admin.site.urls),
+                  path("api/cinema/", include("cinema.urls", namespace="cinema")),
+                  path("api/user/", include("user.urls", namespace="user")),
+                  path("__debug__/", include("debug_toolbar.urls")),
+                  path("api/doc/", SpectacularAPIView.as_view(), name="schema"),
+                  path("api/doc/swagger/", SpectacularSwaggerView.as_view(url_name="schema"), name="swagger-ui"),
+                  path("api/doc/redoc/", SpectacularRedocView.as_view(url_name="schema"), name="redoc"),
+                  path("api/token/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
+                  path("api/token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
+              ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
The `static()` function is used to serve media files during development. Ensure that in production, a proper web server or cloud service is configured to serve static and media files.